src/components/App/App.module.css

```diff
@@ -1,9 +1,13 @@
 .wrap {
     display: block;
     margin: auto;
-    width: 300px;
+    width: 500px;
+    background-color: #ffffff;
+    border-radius: 7px;
+    box-shadow: 0 0 8px rgba(0,0,0,0.3);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5693557)